node/node_test.go

```diff
@@ -52,7 +53,7 @@ func defaultNewTestNode(config *cfg.Config, logger log.Logger) (*Node, error) {
 		proxy.DefaultClientCreator(config.ProxyApp, config.DBDir()),
 		DefaultGenesisDocProviderFunc(config),
 		InMemDBProvider,
-		ipfs.Mock(),
+		ipfs.DagOnlyMock(),
```

Comment:
The full mock node was part of the reason why `TestNodeSetPrivValIPC` kept failing. TBH, I'm still not 100% sure why, as this test was passing in #352, and the mocked IPFS api obejct was only moved to the blockstore. :man_shrugging: